packages/web-components/src/components.d.ts

```diff
@@ -393,6 +393,20 @@ export namespace Components {
          */
         "required"?: boolean;
     }
+    interface VaIcon {
+        /**
+          * the name of the icon to use
+         */
+        "icon": string;
```

Comment:
This component is the first one we've done that just jumps right to USWDS (so there's no `uswds` prop like the others). Was there any requirement to also add support for Font Awesome icons (what VA.gov is currently using)?

Reply:
The [ticket](https://app.zenhub.com/workspaces/platform-design-system-5f8de67192551b0012ebb802/issues/gh/department-of-veterans-affairs/vets-design-system-documentation/1801) doesn't mention it but maybe @Andrew565 can confirm?

Reply:
I believe @humancompanion-usds had said "FontAwesome is a nice-to-have, not a need-to-have", but he can confirm or deny that.